module/metrics/execution.go

```diff
@@ -739,6 +749,11 @@ func (ec *ExecutionCollector) ExecutionTransactionExecuted(
 	ec.transactionExecutionTime.Observe(float64(dur.Milliseconds()))
 	ec.transactionConflictRetries.Observe(float64(numConflictRetries))
 	ec.transactionComputationUsed.Observe(float64(compUsed))
+	if compUsed > 0 {
+		// normalize so the value should be around 1
+		ec.transactionNormalizedTimePerComputation.Observe(
```

Comment:
Yes `EstimatedComputationPerMillisecond` does depend on how we do the calibration. When it changes we have to keep that in mind when comparing old transactions and new transactions.

when comparing transaction performance we should Ideally compare actually transaction execution time over comparing computation, because computation is just a model for execution time.